comfyui_blender/utils.py

```diff
@@ -7,7 +7,7 @@
 import bpy
 
 
-def download_file(filename, subfolder):
+def download_file(filename, subfolder, output_type="output"):
```

Comment:
Rename argument from `output_type` to `type` to align with the websocket message / other arguments.

```suggestion
def download_file(filename, subfolder, type="output"):
```